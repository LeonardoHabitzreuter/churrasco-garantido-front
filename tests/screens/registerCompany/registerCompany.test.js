import React from 'react'
import { shallow } from 'enzyme'
import RegisterCompany from '../../../src/screens/registerCompany'

describe('Register company component', () => {
  test('should renders properly', () => {
    const wrapper = shallow(<RegisterCompany />)
    expect(wrapper).toMatchSnapshot()
  })
})
