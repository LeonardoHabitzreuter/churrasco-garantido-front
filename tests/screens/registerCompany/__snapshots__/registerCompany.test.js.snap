// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register company component should renders properly 1`] = `
<div
  className="col-sm-5"
>
  <PageHeader
    bsClass="page-header"
  >
    Cadastrar empresa
  </PageHeader>
  <Loader
    loading={false}
    size="large"
  />
  <Alert
    handleDismiss={[Function]}
    messages={Array []}
    show={false}
    style=""
  />
  <Form
    buttonName="Cadastrar"
    fields={
      Array [
        Object {
          "labelName": "Nome",
          "maxLength": 30,
          "minLength": 4,
          "name": "name",
          "required": true,
          "type": "text",
          "value": "",
        },
        Object {
          "labelName": "CNPJ",
          "name": "cnpj",
          "type": "text",
          "validation": [Function],
          "value": "",
        },
      ]
    }
    inputSize={11}
    labelSize={1}
    onSubmit={[Function]}
  />
</div>
`;
